fuzzers/fuzzbench_qemu/src/fuzzer.rs

```diff
@@ -196,8 +199,8 @@ fn fuzz(
     println!("Stack pointer = {stack_ptr:#x}");
     println!("Return address = {ret_addr:#x}");
 
-    emu.remove_breakpoint(test_one_input_ptr); // LLVMFuzzerTestOneInput
-    emu.set_breakpoint(ret_addr); // LLVMFuzzerTestOneInput ret addr
+    emu.unset_breakpoint_addr(test_one_input_ptr); // LLVMFuzzerTestOneInput
```

Comment:
You might consider `_at` instead of `_addr` (or, what other kinds of breakpoints are there?)

Reply:
It was to make explicit the difference between the low-level API (`{set,unset}_breakpoint_addr`) and the higher-level one (`{add,remove}_breakpoint`).
It may not be the best choice indeed.